CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # Changelog
 
+## 0.38.0      2021-03-31
+
+* Update to pg_query 2.0 and Postgres 13 parser
+  - This is a major upgrade in terms of supported syntax (Postgres 10 to 13),
+    as well as a major change in the fingerprints, which are now shorter and
+    not compatible with the old format.
+  - When you upgrade to this version of the collector you will see a break
+    in statistics, that is, you will see new query entries in pganalyze after
+    adopting this version of the collector.
```

Comment:
Maybe we should highlight the break more? Like just markdown **you will see a break in statistics** or something?

Reply:
Yeah, good idea.